packages/expo-background-fetch/src/BackgroundFetch.ts

```diff
@@ -55,7 +55,7 @@ export async function setMinimumIntervalAsync(minimumInterval: number): Promise<
  * TaskManager.defineTask(YOUR_TASK_NAME, () => {
  *   try {
  *     const receivedNewData = // do your background fetch here
- *     return receivedNewData ? BackgroundFetch.Result.NewData : BackgroundFetch.Result.NoData;
+ *     return receivedNewData ? BackgroundFetch.BackgroundFetchResult.NewData : BackgroundFetch.BackgroundFetchResult.NoData;
```

Comment:
```suggestion
 *     return receivedNewData ? BackgroundFetchResult.NewData : BackgroundFetchResult.NoData;
```

`BackgroundFetchResult` is imported form the types at the top of file, so we can skip `BackgroundFetch` here.